server/users.go

```diff
@@ -45,20 +57,39 @@ type userResponse struct {
 	Name     string    `json:"name"`
 	Provider string    `json:"provider"`
 	Scheme   string    `json:"scheme"`
+	Roles    []string  `json:"roles"`
 }
 
 func newUserResponse(u *chronograf.User) *userResponse {
+	roles := make([]string, len(u.Roles))
+	for i, r := range u.Roles {
```

Comment:
No need for this if you use []chronograf.Role

Reply:
👍